src/BuildTemplate.ts

```diff
@@ -157,6 +176,13 @@ export default class BuildTemplate {
         // add customization for custom source
         let fileCustomizer: any;
         if (!!this._taskParameters.customizerSource) {
+            let winCustomizer: any;
```

Comment:
Good catch. Just rename it to windowsUpdateCustomizer.

Reply:
done